checkov/common/util/feature_flag.py

```diff
@@ -0,0 +1,85 @@
+import os
+import tempfile
+from pathlib import Path
+
+from checkov.common.models.enums import CheckFailLevel
+from checkov.common.util.consts import DEFAULT_EXTERNAL_MODULES_DIR
+from checkov.common.util.http_utils import normalize_bc_url
+from checkov.common.util.type_forcers import convert_str_to_bool, force_int
+from checkov.version import version
+
+
+class FeatureFlag:
```

Comment:
IMO the naming is somewhat misleading since not all env vars are necessarily used as feature flags (BC_API_URL, BC_ROOT_DIR, CACHE_DIR for example)

Reply:
`EnvVars` / `EnvVarsConfig` sounds more accurate